cases/erc20-transfer-events/sqd/.env.example

```
@@ -1,2 +1,8 @@
 RPC_ENDPOINT="https://rpc.ankr.com/eth"
 DB_PORT=5432
+
+# Sqd's archive (v2.archive.subsquid.io) requires an API key as of 19 May 2026.
+# Create one at https://portal.sqd.dev and set it here (read automatically by
+# the @subsquid archive client). Without it the processor fails with
+# CREDENTIALS_INVALID and indexes nothing.
+SQD_API_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)